docs/how-tos/production_configuration.rst

```diff
@@ -0,0 +1,245 @@
+- Batching is not as well tested (as of Redwood) and may have edge cases until it has been used in production
+
+Recommended for:
+
+This is a reasonable choice for most production use cases for small-to-medium sized production deployments and has been load tested up to significant levels.
+
+
+**Vector**
+
+Vector is a log forwarding service that monitors the logs from docker containers or Kubernetes pods. It writes events directly to ClickHouse and automatically batches events based on volume. The LMS can be configured to transform and log xAPI events in-process and Vector will pick them up by reading the logs.
+
+Pros:
+
+- Removes the need to run or scale Ralph
+- Automatic batching adjustments
+- Fastest delivery times to ClickHouse
+- Vector failures do not impact other systems
+
+Cons:
+
+- It is a new service for most operators
+- Events are not de-duplicated before insert, which can result in some (mostly temporary) incorrect data in a disaster recovery
+- Disaster recovery hasn't been tested with Aspects yet
+- Needs a pod run for every LMS or CMS Kubernetes worker
+- When run in-process, adds a small amount of overhead to any LMS request that sends an xAPI statement
+
+Recommended for:
+
+Resource constrained Tutor local environments experienced operators on larger deployments.
+
+
+**Event Bus (experimental)**
+
+Open edX has had event bus capabilities with redis and Kafka backends since Palm. In Redwood the event-tracking and event-routing-backends libraries have been updated to support `using the event bus`_ as a replacement for Celery. It has the advantage of being able to remove Celery contention and maintain better delivery guarantees while still supporting Ralph deduplication and other advanced use cases (such as real-time streaming xAPI events to other reporting or backup services).
+
+Pros:
+
+- Can use completely separate hardware from Celery, providing better performance domain boundaries
+- Can use the same batching mechanisms as Celery for improved performance, with the same costs and tradeoffs
+- Has better delivery guarantees than Celery (in most configurations), allowing operators to recover from event bus consumer outages or upgrades without having to replay tracking logs
+- Can support advanced use cases for sending the xAPI events to other sinks with the same near real-time delivery guarantees as Aspects
+- (redis) Can scale down to use the same redis instance as edx-platform, or use a separate / hosted redis for better isolation and performance
+- (Kafka) Can scale up to handle extremely high volumes of data and handle long-term outages better than any other option, can be run hosted
+- (Kafka) Can potentially `integrate directly`_ with ClickHouse removing the need for both a separate event consumer and Ralph.
+
+Cons:
+
+- Running consumers is not currently handled automatically by Aspects
```

Comment:
There is a PR that enables the consumers and producers components: https://github.com/openedx/tutor-contrib-aspects/pull/610